aospy/test/test_data_loader.py

```diff
@@ -469,9 +475,17 @@ def test_load_variable(self):
             intvl_in='monthly')
         filepath = os.path.join(os.path.split(ROOT_PATH)[0], 'netcdf',
                                 '00050101.precip_monthly.nc')
-        expected = xr.open_dataset(filepath)['condensation_rain']
+        expected = _open_ds_catch_warnings(filepath)['condensation_rain']
         np.testing.assert_array_equal(result.values, expected.values)
 
+    def test_load_variable_does_not_warn(self):
+        with warnings.catch_warnings(record=True) as warnlog:
+            self.data_loader.load_variable(condensation_rain,
+                                           datetime(5, 1, 1),
+                                           datetime(5, 12, 31),
+                                           intvl_in='monthly')
+        assert len(warnlog) == 0
```

Comment:
It looks like we'll need to address the `FutureWarning` in the latest version of xarray for this test to pass:
```
  /home/travis/build/spencerahill/aospy/aospy/utils/times.py:475: FutureWarning: xarray.DataArray.__contains__ currently checks membership in DataArray.coords, but in xarray v0.11 will change to check membership in array values.
    if RAW_START_DATE_STR in da:
```
I think we'll just need to change: https://github.com/spencerahill/aospy/blob/82af0ba36771bb4fd183e2d7460c278dcc45d9bb/aospy/utils/times.py#L473
to:
```python
if RAW_START_DATE_STR in da.coords:
```